src/components/stationsDropDown.tsx

```diff
@@ -0,0 +1,33 @@
+// Currently unimplemented - delete soon
+import React, { Dispatch, ReactElement, SetStateAction } from 'react';
+import Select, { SingleValue } from 'react-select';
+import stationOptions from '../helpers/stationOptions';
+
+type Props = {name: string; id: string; valueSetter: Dispatch<SetStateAction<string>>}
+
+const StationsDropDown: React.FC<Props> = (props):ReactElement => {

```

Comment:
you can break down props immediately here, i.e. 
```suggestion
const StationsDropDown: React.FC<Props> = ({ name, id, valueSetter }):ReactElement => {

```
and reference these in the component without prefixing props.